node/actors/bft/src/lib.rs

```diff
@@ -93,7 +94,19 @@ impl Config {
                 let InputMessage::Network(req) = pipe.recv.recv(ctx).await?;
                 use validator::ConsensusMsg as M;
                 match &req.msg.msg {
-                    M::ReplicaPrepare(_) | M::ReplicaCommit(_) => leader_send.send(req),
+                    M::ReplicaPrepare(_) => {
+                        // This is a hacky way to do a clone. This is necessary since we don't want to derive
+                        // Clone for ConsensusReq. When we change to ChonkyBFT this will be removed anyway.
```

Comment:
this is hacky indeed. The right way would be to assign a fresh ack channel to each of the copies and spawn a background task to wait for both of them to get processed, and then ack the received channel.

Reply:
Agree that it's hacky, but shouldn't cause any problems, correct?

Reply:
it shouldn't break anything, but it escapes inflight request limiting

Reply:
We only have this until ChonkyBFT, so it will only be for PoA single leader.